### ISSUE TITLE
CORS preflight

### ISSUE THREAD

**Issue by aigoncharov** [typestack/routing-controllers#388]
Routing-controllers allows to set config for 'cors' module, but it doesn't enabling handling of pre-flight requests https://github.com/expressjs/cors#enabling-cors-pre-flight
I suggest to expose another option to createExpressServer (like 'handleCorsPreflight') to enable it.

**Comment by Zenulous** [comment]
I agree that it would be nice to have an intuitive solution. Luckily, it is possible to handle preflight as follows:

```js
cors: { preflightContinue: false }```

This will enable CORS and handle preflight without passing preflight requests onto further middleware, ensuring that both 'types' of CORS requests can succeed.
